python/cuspatial/cuspatial/io/geopandas_reader.py

```diff
@@ -30,6 +30,10 @@ def parse_geometries(geoseries: gpGeoSeries) -> tuple:
     polygon_offsets = [0]
 
     for geom in geoseries:
+        if geom is None:
+            all_offsets.append(-1)
```

Comment:
Should we define -1 as a constant?

Reply:
Sure, I can just use `Feature_Enum.NONE.value`

Reply:
Replaced it with a simple constant.